### ISSUE TITLE
[BUG] Being webbed by spiders causes extreme lag spikes

### ISSUE THREAD

**Issue by Eksekk** [Rebirth-of-the-Night/Rebirth-Of-The-Night#292]
ISSUES NOT USING OUR TEMPLATE WILL NOT BE CONSIDERED AND WILL BE CLOSED!

**Describe the bug**
When you are afflicted with Slowness V from spider attacks and they web you, you get extreme lag spikes. They make it almost impossible to fight back, and even if one web breaks another is created almost immediately. (If this cannot be fixed, maybe remove Weaker Cobwebs, as it makes game harder rather than easier?)

**To Reproduce**
1. Let spider attack you until you are webbed.
2. Observe your FPS.

**Expected behavior**
No lag.

**Screenshots**
N/A

**Info (please complete the following information):**
 - Modpack Version: 2.77.1
 - Server or Singleplayer: Singleplayer
 - Version the world was created on: 2.77.1

**Additional context**
Almost certainly caused by Weaker Cobwebs, before its addition there was no such issue.

**Comment by SandwichHorror** [comment]
Can't replicate this on singleplayer.

**Comment by Eksekk** [comment]
Alright, so I will try to get rid of the problem on my end and if I do manage, I will write here what helped, otherwise I will just close this issue for now (I could upload a video, but that probably wouldn't help to solve it).

**Comment by CalaMariGold** [comment]
> maybe remove Weaker Cobwebs, as it makes game harder rather than easier?

What? The whole point of that mod is to nerf spiders.

**Comment by kil0byt3** [comment]
@CalaMariGold I believe Eksekk means "makes it harder", as in, the lagspikes make it unplayable for them

**Comment by Eksekk** [comment]
Yes, @kil0byt3 said what I meant to.

Tests came out inconclusive. I tried using LagGoggles (nothing suspicious) and installing fresh version (because I was playing with my own anvil mod). On modified version lag was constantly, while on fresh it occurred in the first minutes of the game, then nothing (until I reloaded world). Maybe that's because of higher mod count. Idk if this issue should get closed or not.
